pkg/tsdb/prometheus/querydata/exemplar/framer.go

```diff
@@ -64,8 +64,14 @@ func (f *Framer) Frames() (data.Frames, error) {
 		timeField.Append(b.Timestamp)
 		valueField.Append(b.Value)
 		for i, labelName := range labelNames {
-			labelValue, ok := b.Labels[labelName]
-			if !ok {
+			var labelValue string
+			for _, f := range b.Fields {
+				if f.Name == labelName {
+					labelValue = f.CopyAt(b.RowIdx).(string)
```

Comment:
@itsmylife could an empty string be a valid `labelValue` here? If so we have to change the `if labelValue == ""` check after the loop.

Reply:
No, I do not think empty string is a valid value. @toddtreece any thoughts on this?

Reply:
Thanks @toddtreece - I will change the `if labelValue == ""` check.

Reply:
Done, PTAL.